swmm-toolkit/src/swmm/toolkit/shared_enum.py

```diff
@@ -447,6 +450,8 @@ class NodePollutant(Enum):
     :attr:`~QUALITY`
     """
     QUALITY = 0
+    INF_CONC = 1
```

Comment:
Make more descriptive - INFLOW_CONC

Reply:
Thanks @karosc for making these changes! We also need to add the reactor concentration -- we added both inflow concentration and reactor concentration to SWMM:
    SM_NODECIN        = 1,  /**< Inflow Concentration */
    SM_NODEREACTORC   = 2   /**< Reactor Concentation */
See the old pull request for more details: https://github.com/OpenWaterAnalytics/Stormwater-Management-Model/pull/326.

Reply:
Yes, we should add tests, but I haven't thought about how to design the tests yet. I still have not actually used the new setters in any context. Should we just start a simulation with pollutants at 0, set high concentrations at runtime, then check that they are not zero after a few time steps? What would be your approach here?

Reply:
I haven't exactly figured out the best way to do this either. I didn't write the tests for these additions when we edited SWMM, @abhiramm7 did. 
- Here is how inflow concentration was tested: https://github.com/OpenWaterAnalytics/Stormwater-Management-Model/blob/76b8ae7e39f07a659b51284effafe668394d7f29/tests/solver/test_pollut.cpp#L151-L183
- Here is how reactor concentration was tested: https://github.com/OpenWaterAnalytics/Stormwater-Management-Model/blob/76b8ae7e39f07a659b51284effafe668394d7f29/tests/solver/test_pollut.cpp#L185-L222
Couldn't find where hrt was tested. Not sure if this helps of now? Hopefully @abhiramm7 has some ideas!